packages/interactive_media_ads/lib/src/platform_interface/interactive_media_ads_platform.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'platform_ad_display_container.dart';
+import 'platform_ads_loader.dart';
+import 'platform_ads_manager_delegate.dart';
+
+/// Interface for a platform implementation of the Interactive Media Ads SDKs.
+abstract base class InteractiveMediaAdsPlatform {
+  /// The instance of [InteractiveMediaAdsPlatform] to use.
+  ///
+  /// Platform-specific plugins should set this with their own platform-specific
+  /// class that extends [InteractiveMediaAdsPlatform] when they register
+  /// themselves.
+  static InteractiveMediaAdsPlatform? instance;
+
+  /// Creates a new [PlatformAdsLoader].
+  PlatformAdsLoader createPlatformAdsLoader(
+    PlatformAdsLoaderCreationParams params,
+  ) {
+    throw UnimplementedError(
```

Comment:
Since it's `abstract`, and there are not yet any implementations to be compatible with, should we just remove the implementations?

I'm fine if you want to leave them for consistency with later additions, but I don't think they should be necessary unless these are optional.